v1.0: accepted os versions to run the Meilisearch binary (#2112)
meilisearch/documentation: learn/getting_started/supported_os.md

# Supported operating systems

> Please ensure your operating system is listed on this page before installing the Meilisearch binary. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.

Reviewer: Maybe give link about how to install meilisearch binaries?

> If the provided binaries don't work for you, we recommend compiling Meilisearch from the source files. If the compilation fails, Meilisearch is not compatible with your machine.

Reviewer: And docker?

> If the provided binaries don't work for you, we recommend compiling Meilisearch from the source files. If the compilation fails, Meilisearch is not compatible with your machine.

Author: Same as [previous comment](https://github.com/meilisearch/documentation/pull/2112#discussion_r1081140455)

::: warning

> Any operating system or version not listed on this page does not mean Meilisearch won't work with it at all, we just don't guarantee it.

Reviewer: ```suggestion
Any operating system or version not listed on this page does not mean Meilisearch won't work with it at all, we just don't guarantee it through our test suite.
```

:::

## Linux

> The Meilisearch binary works on all Linux distributions with `amd64/x86_64` or `aarch64/arm64` architecture using glibc 2.27 and later.

Reviewer: Maybe give the command to know the glibs version (I put in the engine team issue)?

The Meilisearch binary works with macOS 12 and later with `amd64` or `arm64` architecture.

## Windows

The Meilisearch binary works with Windows 11 and later.

> If the provided [binaries](/learn/getting_started/quick_start.md#local-installation) don't work for you, try building Meilisearch from source. If compilation fails, Meilisearch is not compatible with your machine.

Reviewer 2: Ok, I should have tested this on the preview before making my suggestion. Now it looks like it's part of the Windows instructions. I think the simplest solution is to return it to where it was, right below the first paragraph. Sorry 😅

# Supported operating systems

Please ensure your operating system is listed on this page before installing the Meilisearch binary. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.

> If the provided [binaries](/learn/getting_started/quick_start.md#local-installation) don't work for you, we recommend compiling Meilisearch from the source files. If the compilation fails, Meilisearch is not compatible with your machine.

Reviewer 2: Besides the minor edits, I would also recommend moving this paragraph so it's after the supported OS list. This way users get the list ASAP, and then only need to read what to do if something goes wrong/their OS is not listed.

```suggestion
If the provided [binaries](/learn/getting_started/quick_start.md#local-installation) don't work for you, try building Meilisearch from source. If compilation fails, Meilisearch is not compatible with your machine.
```

> ::: warning
> Any operating system or version not listed on this page does not mean Meilisearch won't work with it at all, we just don't guarantee it through our test suite.
> :::
> 

Reviewer 2: I tried to integrate the info here on the first paragraph. The tone here is too informal for technical documentation, IMHO.

```suggestion
```
